homeassistant/components/climate/__init__.py

```diff
@@ -43,6 +43,9 @@
 STATE_AUTO = "auto"
 STATE_DRY = "dry"
 STATE_FAN_ONLY = "fan_only"
+STATE_AWAY_HOLD = "away"
+STATE_HOME_HOLD = "home"
+STATE_TEMPERATURE_HOLD = "temp"
```

Comment:
@nordlead2005 Actually, in ecobee terminology, "home" and "away" are **schedules**. In hass they hey are the state of the mode variable. Then there is "smart home" and "smart away" (and others) which are "hold states". The latter is this what the constants above refer to. "Away" is already used by a number of other thermostats. I would think the others would end up being implemented also in others that support different hold modes. 

@MartinHjelmare I guess that depends on what you think of as the state of the thermostat. Obviously, anything that is a property of the thermostat that can change and affects its behavior is part of its state in a broad sense. However, you may have a much narrower sense, that considers only some attributes to define the state, e.g., operation mode ("heat", "cool") for example.

I was trying to follow a pattern where folks seem to define such constants to achieve consistency between devices (I guess). I can take this out again, as it is just cosmetic.